help.go

```diff
@@ -28,7 +28,9 @@ COMMANDS:
    {{end}}{{if .Flags}}
 GLOBAL OPTIONS:
    {{range .Flags}}{{.}}
-   {{end}}{{end}}
+   {{end}}{{end}}{{if .Copyright }}
+COPYRIGHT:
+   {{.Copyright}}{{end}}
 `
```

Comment:
An observation, the above could be rewritten to make it a little simpler by using `with` [action](https://godoc.org/text/template#hdr-Actions) instead of `if`:

```
{{with .Copyright}}
COPYRIGHT:
   {{.}}{{end}}
```